content/03.methods.md

```diff
@@ -260,21 +260,16 @@ $$
 These results do not include signatures with small contributions; deconstructSigs drops signature weights that are less than 6% [@doi:10.1186/s13059-016-0893-4].
 We used these methods to calculate signature scores for each sample with both COSMIC [@url:https://cancer.sanger.ac.uk/cosmic] and Alexandrov et al, 2013 [@doi:10.1038/nature12477] signature sets.
 
-### PBTA Tumor Mutation Burden
+### PBTA and TCGA Tumor Mutation Burden
 
 We consider tumor mutation burden (TMB) to be the number of consensus SNVs per *effectively surveyed* base of the genome.
 
 $$ \textrm{TMB} = \frac{\textrm{\# of coding sequence SNVs}}{\textrm{Size in Mb of {\em effectively surveyed} genome}}
 $$
 
-We used the total number coding sequence consensus SNVs for the numerator and the size of the intersection of the regions considered by Lancet, Strelka2, and Mutect2 with coding regions (CDS from GENCODE v27 annotation [@url:https://www.gencodegenes.org/human/release_27.html]) as the denominator.
+We used the total number coding sequence consensus SNVs for the numerator and the size of the intersection of the regions considered by Strelka2 and Mutect2 with coding regions (CDS from GENCODE v27 annotation [@url:https://www.gencodegenes.org/human/release_27.html]) as the denominator.
 
-### TCGA Tumor Mutation Burden
-
-We calculated tumor mutation burden in TCGA using MC3 mutation calls [@doi:10.1016/j.cels.2018.03.002] for TCGA brain-related tumor projects including: LGG (lower-grade glioma) [@doi:10.1056/NEJMoa1402121], GBM (glioblastoma multiforme) [@doi:10.1016/j.cell.2013.09.034], and PCPG (pheochromocytoma and paraganglioma) [@doi:10.1016/j.ccell.2017.01.001].
-The MC3 project provided an exome BED file.
-All SNVs fell within these regions.
-We considered the regions covered by the MC3 BED file (based on GENCODE v19 annotation [@url:https://www.gencodegenes.org/human/release_19.html]) to have been effectively surveyed.
+Lancet called particularly low VAF mutations for WXS sample data, so it was dropped from TMB calculations because TCGA data is all WXS samples [@url: https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/snv-callers/lancet-wxs-tests].
```

Comment:
```suggestion
Lancet showed a bias for calling mutations with a much lower VAF for WXS than WGS data (**Figure S2, panel G**).
Since TCGA data was all WXS, we dropped Lancet from TMB calculations.  [@url: https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/snv-callers/lancet-wxs-tests].
```

I am going to add this to supplemental figure 2 and add a snippet for discussion in the results because I think it is worth showing.